composer.json

```diff
@@ -25,7 +25,7 @@
     "extra": {
         "laravel": {
             "providers": [
-                "Webkul\\Dropship\\Providers\\DropshipServiceProvider"
+                "Webkul\\Dropship\\DropshipServiceProvider"
```

Comment:
Please restore this back to `"Webkul\\Dropship\\Providers\\DropshipServiceProvider"`